### ISSUE TITLE
image alt text

### ISSUE THREAD

**Issue by evanwill** [drbutterfield/theory-draft-1#3]
future enhancement:  all images should have `alt` text describing the contents of the image for accessibility and usability, it appears when the image can't be loaded and used by screen readers. 

in markdown the `alt` is set in square brackets, for example `![alt text](image/src)`

in html use the alt attribute, for example `<img alt="alt text" src="image/src">`
